numismatic/feeds/base.py

```diff
@@ -93,21 +94,27 @@ def get_prices(self, assets, currencies, raw=False):
     def get_tickers(self, assets, currencies, raw=False):
         return
 
-    @classmethod
-    def subscribe(cls, assets, currencies, channels):
-        assets = cls._validate_parameter('assets', assets)
-        currencies = cls._validate_parameter('currencies', currencies)
-        channels = cls._validate_parameter('channels', channels)
+    def subscribe(self, assets, currencies, channels, exchange=None,
+                  interval=1.0):
```

Comment:
Is interval in seconds? If so, perhaps we can call it `interval_seconds`

Reply:
Yes it is in seconds